src/app/onboarding-complete/page.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import Link from 'next/link';
+import { LogoImage, OuterDiv } from '@/app/onboarding/styles';
+import { BigLinkButton } from '@/components/Buttons';
+import CONFIG from '@/lib/configs';
+import COLORS from '@/styles/colors';
+import { CenteringDiv, Flex, SmallCard } from '@/styles/containers';
+import { H2Centered, P } from '@/styles/text';
+import IJPLogo from '~/public/images/ijp_logo_blue.webp';
+
+export default function Page() {
+  return (
+    <OuterDiv>
+      <Link href={CONFIG.homepage}>
+        <LogoImage $show width="205" alt="IJP Logo" src={IJPLogo} />
+      </Link>
+      <CenteringDiv>
```

Comment:
is it necessary to have a CenteringDiv nesting a Flex centered/column ?

Reply:
i think so, the `CenteringDiv` horizontally centers the `Flex`, which vertically centers the card